### DISCUSSION TITLE
Feedback on connection screen

### DISCUSSION THREAD

**Discussion by BenGWeeks** [ZeusLN/zeus#958] [Ideas]
Hi there,

Would it be possible to give feedback to the user on the connection screen that things are happening?

![20220422_151741000_iOS](https://user-images.githubusercontent.com/5950171/164745502-868db061-430d-4645-a6b0-ea5180507d11.png)

For example:

```
Connecting to Tor ... connected
Connecting to node ... connected
```

The more verbose the better :-)

Many thanks.

**Comment by BenGWeeks** [comment]
Also consider caching content then refreshing node in the background (and notifications do not block the UI).

**Comment by Bosch-0** [comment]
Good suggestion, will get on to this for a future update.

If the cache the content too we could also avoid this loading screen all together and rather have the user open straight into the home screen and show a 'connecting' icon in the header section. The state shown will be the last update that that was cached.
